### PR TITLE
HCAP-1186 | Enable Feature Flag for multi org | Revert some unwanted changes

### PR DESCRIPTION
**Pull request by fwpushan**
- [x] Add service layer for feature flag
- [x] Add feature flag for Multi-org 
- [x] Update Client code to support feature flagging
- [x] Revert some unwanted API code (feature flagging is difficult)

Ticket: [HCAP-1186](https://freshworks.atlassian.net/browse/HCAP-1186)

### CODE REVIEW

**Review by taraepp — DISMISSED** [review]
I've got a couple questions but don't see any problems

File: server/tests/participant-status.test.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const {
 
 const regions = ['Fraser', 'Interior', 'Northern', 'Vancouver Coastal', 'Vancouver Island'];
 
-describe('Test Participant status data model and service', () => {
+describe.skip('Test Participant status data model and service', () => {
```
[the remark this answers is not in the document]
Comment (fwpushan) [reply server/tests/participant-status.test.js:18]: Those tests are regarding Multi-org. So disabling for now

File: cypress/fixtures/participantTableTabs.json [hunk 2]
```diff
@@ -50,7 +50,7 @@
     {
       "name": "health_authority",
       "fixture": "test-ha",
-      "tabsWithMultiselect": ["Available Participants"],
```
[the remark this answers is not in the document]
Comment (fwpushan) [reply cypress/fixtures/participantTableTabs.json:53]: This is fixture data, no way we can put any logic here. Based on the flag we can load different fixtures.   what do you suggest?

[the remark this answers is not in the document]
Comment (alva-fresh) [reply cypress/fixtures/participantTableTabs.json:53]: Might be a good idea to load different fixtures based on certain flag

File: server/migrations/1649795536386_create-site-participants-status-table.js [hunk 3]
```diff
@@ -1,31 +0,0 @@
-/* eslint-disable camelcase */
-const { collections } = require('../db');
```
[the remark this answers is not in the document]
Comment (fwpushan) [reply server/migrations/1649795536386_create-site-participants-status-table.js:2]: Dev/test only. Already reverted it from dev. Will do the same before deployment to test.

File: cypress/fixtures/participantTableTabs.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (fwpushan) [reply cypress/fixtures/participantTableTabs.json]: @alva-fresh I added the feature flag support for test execution but not on loading of the fixtures. To load fixtures based on the feature flag, we may call some additional API that could be performance overhead.

**Review by taraepp — APPROVED** [review]
Looks good

File: cypress/integration/participantTable.spec.js [hunk 4]
```diff
@@ -15,14 +15,18 @@ describe('Participant Table', () => {
     it(`Correctly renders columns for ${role.name}`, () => {
       cy.kcLogin(role.fixture);
       cy.visit('/participant-view');
+      const featureMultiOrgProspecting =
+        localStorage.getItem('FEATURE_MULTI_ORG_PROSPECTING') !== undefined
```
Comment (alva-fresh) [inline cypress/integration/participantTable.spec.js:19]: Can this be changed to something like this?
```
const featureMultiOrgProspecting =  localStorage.getItem('FEATURE_MULTI_ORG_PROSPECTING') || false;
```

Comment (fwpushan) [reply cypress/integration/participantTable.spec.js:19]: That will be a fallacy.

**Review by alva-fresh — APPROVED** [review]

**Pull request merged**